frontend/src/store/modules/visitors/actions.js

```diff
@@ -9,11 +24,31 @@ export default {
         context.commit(SET_ACTIVE_BUTTON, button);
     },
     [CHANGE_FETCHED_BUTTON_STATE]: (context, data) => {
-
         if (data.value) {
             context.commit(SET_BUTTON_FETCHING, data.button);
         } else {
             context.commit(RESET_BUTTON_FETCHING, data.button);
         }
     },
+    [GET_BUTTON_DATA]: (context, data) => {
+        if (data.value) {
```

Comment:
do not wrap logic in block, it increses complexity of code. Better do

if (!data.value) {
    return;
}